src/main/kotlin/org/jitsi_modified/impl/neomedia/rtp/sendsidebandwidthestimation/SendSideBandwidthEstimationConfig.kt

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jitsi_modified.impl.neomedia.rtp.sendsidebandwidthestimation
+
+import org.jitsi.config.JitsiConfig
+import org.jitsi.config.legacyProperty
+import org.jitsi.config.newProperty
+import org.jitsi.config.simple
+import org.jitsi.nlj.util.Bandwidth
+import org.jitsi.utils.config.dsl.multiProperty
+import org.jitsi.utils.config.dsl.property
+
+class SendSideBandwidthEstimationConfig {
+    companion object {
+        private const val LEGACY_BASE_NAME =
+            "org.jitsi.impl.neomedia.rtp.sendsidebandwidthestimation.sendsidebandwidthestimation"
+
+        /**
+         * The property that specifies the low-loss threshold
+         * (expressed as a proportion of lost packets) when the loss probability
+         * experiment is *not* active
+         */
+        private val defaultLowLossThresholdProp = property<Double> {
+            readOnce()
+            name("jmt.bwe.send-side.low-loss-threshold")
+            fromConfig(JitsiConfig.newConfig)
+        }
+
+        @JvmStatic
+        fun defaultLowLossThreshold(): Double = defaultLowLossThresholdProp.value
+
+        /*
+         * The property that specifies the high-loss threshold
+         * (expressed as a proportion of lost packets) when the loss probability
+         * experiment is *not* active.
+         */
+        private val defaultHighLossThresholdProp = property<Double> {
+            readOnce()
+            name("jmt.bwe.send-side.high-loss-threshold")
+            fromConfig(JitsiConfig.newConfig)
+        }
+
+        @JvmStatic
+        fun defaultHighLossThreshold(): Double = defaultHighLossThresholdProp.value
+
+        /**
+         * The property that specifies the bitrate threshold (in kbps) when the
+         * loss probability experiment is *not* active.
+         */
+        private val defaultBitrateThresholdProp = property<Bandwidth> {
+            name("jmt.bwe.send-side.bitrate-threshold")
+            readOnce()
+            fromConfig(JitsiConfig.newConfig)
+            retrievedAs<String>() convertedBy(Bandwidth.Companion::fromString)
+        }
+
+        @JvmStatic
+        fun defaultBitrateThresholdKbps(): Int = defaultBitrateThresholdProp.value.kbps.toInt()
```

Comment:
The only place this is used converts it back to bps. Why not export it as bps?

Reply:
my plan was to go back and change the class to use it as `Bandwidth` directly, but i was saving that for a bigger refactor, so i was just minimizing the change by matching the old value.